beets/importer.py

```diff
@@ -50,7 +50,18 @@
 SINGLE_ARTIST_THRESH = 0.25
 PROGRESS_KEY = 'tagprogress'
 HISTORY_KEY = 'taghistory'
-# Album and item flexble attrbutes that should not be preserved on reimports.
+"""
+Album and item flexible attributes that should not be preserved on reimports.
+All other fields will be preserved (i.e., not updated) during reimports. If
+you prefer to change this behavior, you can modify the
+`REIMPORT_FRESH_FIELDS_ALBUM` and the REIMPORT_FRESH_FIELDS_ITEM` lists in
+your code. For example, if you want to allow updating update a flexible item
+attribute `tidal_track_popularity`, you can add it to the list like this:
+
+from beets import importer
+def extend_reimport_fresh_fields_item():
+    importer.REIMPORT_FRESH_FIELDS_ITEM.extend(['tidal_track_popularity'])
+"""
```

Comment:
I tried to rephrase and shorten this a comment a little but it seems that I'm having a hard time. Maybe you can work from here:

```
# Usually flexible attributes are preserved (i.e., not updated) during
# reimports. The following two lists (globally) change this behaviour for
# certain fields. To alter these lists only when a specific plugin is in use,
# something like this can be used within that plugin's code:
#
# from beets import importer
# def extend_reimport_fresh_fields_item():
#     importer.REIMPORT_FRESH_FIELDS_ITEM.extend(['tidal_track_popularity']
# )
```

Reply:
That revision actually looks pretty good to me!